src/app/services/actions/permissions-action-creator.spec.ts

```diff
@@ -432,8 +438,8 @@ describe('Permissions action creators', () => {
         {
           type: 'QUERY_GRAPH_STATUS',
           response: {
-            statusText: 'Failed',
-            status: 'An error occurred when unconsenting. Please try again',
+            statusText: 'Unable to dissent',
+            status: 'Unable to dissentYou require the following permissions to revoke',
```

Comment:
Nit: Space between `dissent.` and `You`

Reply:
The test fails if I add the space because that's how the error is being passed